redfish/job.go

```diff
@@ -0,0 +1,131 @@
+package redfish
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/stmcginnis/gofish/common"
```

Comment:
This will need to be moved down into it's own "local" import group to follow the linter rules.